sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/function/impl/PopFunction.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.service.function.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import io.smallrye.mutiny.Uni;
+import io.stargate.bridge.proto.Schema;
+import io.stargate.sgv2.docsapi.api.exception.ErrorCode;
+import io.stargate.sgv2.docsapi.api.exception.ErrorCodeRuntimeException;
+import io.stargate.sgv2.docsapi.service.function.BuiltInFunction;
+import io.stargate.sgv2.docsapi.service.query.ReadDocumentsService;
+import io.stargate.sgv2.docsapi.service.write.WriteDocumentsService;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+/** Pop function implementation. Pops a value to the existing array. Returns the popped value. */
+@ApplicationScoped
+public class PopFunction implements BuiltInFunction {
+
+  @Inject ReadDocumentsService readBridgeService;
```

Comment:
There's probably reason for this but... why isn't name `readDocumentsService`, similar to its Write-counterpart?